docs/resources/service-providers.md

```diff
@@ -4,79 +4,194 @@ order: 5
 
 # Service Providers
 
-We define 'service providers' as entities providing services for end-users that involve some form of interaction with a Cosmos-SDK based blockchain (this includes the Cosmos Hub). More specifically, this document will be focused around interactions with tokens.
+'Service Providers' are defined as entities that provide services for end-users that involve some form of interaction with the Cosmos Hub. More specifically, this document is focused on interactions with tokens.
 
-This section does not concern wallet builders that want to provide Light-Client functionalities. Service providers are expected to act as trusted point of contact to the blockchain for their end-users. 
+Service Providers are expected to act as trusted points of contact to the blockchain for their end-users. This Service Providers section does not apply to wallet builders that want to provide Light Client functionalities. 
 
-## High-level description of the architecture
+This document describes:
 
-There are three main pieces to consider:
+- [Connection Options](#connection-options)
+- [Running a Full Node](#running-a-full-node)
+  - [What is a Full Node?](#what-is-a-full-node)
+  - [Installation and Configuration](#installation-and-configuration)
+- [Command-Line Interface](#command-line-interface)
+  - [Available Commands](#available-commands)
+  - [Remote Access to gaiad](#remote-access-to-gaiad)
+  - [Create a Key pair](#create-a-key-pair)
+  - [Check your Account](#check-your-account)
+  - [Check your Balance](#check-your-balance)
+  - [Send coins using the CLI](#send-coins-using-the-cli)
+- [REST API](#rest-api)
+  - [Listen for incoming transactions](#listen-for-incoming-transaction)
 
-- Full-nodes: To interact with the blockchain. 
-- Rest Server: This acts as a relayer for HTTP calls.
-- Rest API: Define available endpoints for the Rest Server.
 
-## Running a Full-Node
+## Connection Options
 
-### Installation and configuration
+There are four main technologies to consider to connect to the Cosmos Hub:
 
-We will describe the steps to run and interact with a full-node for the Cosmos Hub. For other SDK-based blockchain, the process should be similar. 
+- Full Nodes: Interact with the blockchain. 
+- REST Server: Serve as a relayer for HTTP calls.
```

Comment:
Suggest changing the term `relayer` as it overlaps with IBC's relayer terminology.